internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -112,11 +119,27 @@ func convertSidecar(s map[string]*manifest.SidecarConfig) ([]*template.SidecarOp
 			EntryPoint:   entrypoint,
 			HealthCheck:  convertContainerHealthCheck(config.HealthCheck),
 			Command:      command,
+			PortMappings: sidecarPortMappings,
 		})
 	}
 	return sidecars, nil
 }
 
+// ConvertPortMapping return the port mapping object based on the given input.
+func ConvertPortMapping(exposedPorts []manifest.ExposedPort) []*template.PortMapping {
```

Comment:
Why do we make this a public func? Also this signature looks odd. I would expect

```suggestion
func ConvertPortMapping(exposedPort manifest.ExposedPort) *template.PortMapping {
```

otherwise most likely we are gonna loop twice.

Reply:
This function is public because it being called from LBWS, Backend, Worker service and Scheduled jobs. Also it should be list because we are looping over it to convert all the exposed ports to port mappings.